src/scripts/cronjobs/emailBreachAlerts.ts

```diff
@@ -121,6 +121,16 @@ export async function poll(
 
     const { breachName, hashPrefix, hashSuffixes } = data;
     const breachAlert = getBreachByName(breaches, breachName);
+    // Check added to old code for type safety, but we've been assuming
+    // getBreachByName will always find a breach here without tests so far, so
+    // apparently that's been working well enough:
+    /* c8 ignore next 6 */
+    if (!breachAlert) {
```

Comment:
Technically a change in runtime behaviour, but this would presumably have caused an error before as well, just without the log.